Adds new Meetup Organizers Toolkit (#9)
ansible-community/meetup: milestones/start_planning.md

> Planning is key to a successful event, so lets go through some of the things you'll want to think about.

Reviewer: ```suggestion
Planning is key to a successful event, so let's go through some of the things you'll want to think about.
```

## Mode of the meetup

There are two main ways to conduct a meetup, Online and In-person.

### Online Meetup

> The following is a checklist for having an online meetup
> 
>     - Platform to host the meetup
>     - Fix a date and time
>     - Speaker outreach
>     - Social media promotion plans
>     - Platform to have a conversation with the attendees, speakers, and organizers

Reviewer: ```suggestion
Before you host an online meetup, you should:

    - Select an online platform to host the meetup
    - Fix a date and time
    - Find one or more speakers
    - Promote the event on social media
    - Select a platform to have a conversation with the attendees, speakers, and organizers
```

An online meetup allows you to connect to a broader audience living in different time zones along with the regular meetup group members.  The organizers may want to promote the event on the social media platforms of the time zones coinciding with the event timing to extend the reach of the group.

### In Person meetup

> The following is a checklist for having a in person meetup
> 
>     - Venue to host the meetup
>     - Fix a date and time for the meetup
>     - Speaker outreach
>     - Sponsorship Scout
>     - Social media promotion plans
>     - Platform to have a conversation with the attendees, speakers, and organizers

Reviewer: ```suggestion
Before you host an in-person meetup, you should:

    - Select a venue to host the meetup
    - Fix a date and time for the meetup
    - Find one or more speakers
    - If possible, find one or more sponsors
    - Promote the event on social media
    - Select a platform to have a conversation with the attendees, speakers, and organizers
```

> ## Finding speaker

Reviewer: ```suggestion
## Finding speakers
```

> Selecting the topics forms the core of any meetup group. Before looking for speakers, if the organizer pays heed to these points, that will be helpful.

Reviewer: ```suggestion
Selecting the topics forms the core of any meetup group. As an organizer, you should ask a few questions before you start looking for speakers. For example:
```

What kind of audience are you expecting in the meetup? What is their experience with Ansible and with Automation in general?

> Here are a few place for speaker outreach :

Reviewer: ```suggestion
Here are a few places for speaker outreach :
```

Author: Updated.

      1. Social media, email
      2. Community events
      3. Github contributions
      4. Ansible Community Communication platform

> While reaching out to the speakers, one has to be careful to refrain from spamming people to give talks at the meetups. If you know the person or someone who can introduce them, that would be wonderful. And if not, a personal touch to the message is necessary for the person to know you, your intention, and your ideas, so they can make a reasoned decision about accepting or declining the invite. Here are some template messages which might be helpful in this process.

Reviewer: ```suggestion
While reaching out to the speakers, be careful to refrain from spamming people to give talks at the meetups. If you know the person or someone who can introduce them, that would be wonderful. And if not, a personal touch to the message is necessary for the person to know you, your intention, and your ideas, so they can make a reasoned decision about accepting or declining the invite. Here are some template messages which might be helpful in this process.
```
I don't see any templates here yet . . .

Author: Added this line 'There are some [template messages](./resources/email_templates) in the resource section which might be helpful in this process.'

## Planning tools

In addition to the meetup.com group, we can create a repo for your meetup group on https://github.com/ansible-community/

You can use the repo to:

1. Get talk ideas and proposals for upcoming meetups from your meetup group
2. Upload and share presentation slides / code samples from speakers
3. Create an issue per event to gather feedback from attendees

## Session ideas

1. What is Ansible?
2. Ansible 101
3. Writing your first playbook
4. How to contribute to Ansible?
5. From Playbook to Rulebook (presentaion for EDA)
6. Ansible Ecosystem

> ## Venue Scout

Reviewer: ```suggestion
## Finding a Venue
```

Author: Updated.

Any in-person event can not happen without a venue. There are some templates in the resources  section which might be helpful in the journey of venue scout.

> ## Sponsor Scout

Reviewer: ```suggestion
## Finding Sponsors
```
